docs/index.md

```diff
@@ -1,27 +1,279 @@
+
+// Collection clients do not require a parameter.
+const actorCollectionClient = apifyClient.actors();
+// Creates an actor with the name: my-actor.
+const myActor = await actorCollectionClient.create({ name: 'my-actor-name' });
+// List all your used Actors (both own and from Apify Store)
+const { items } = await actorCollectionClient.list();
+```
+
+
+:::note Resource identification
+
+The resource ID can be either the `id` of the said resource, or a combination of your `username/resource-name`.
+
+:::
+
+
+```js
+const { ApifyClient } = require('apify-client');
+
+const apifyClient = new ApifyClient({ token: 'MY-APIFY-TOKEN' });
+
+// Resource clients accept an ID of the resource.
+const actorClient = apifyClient.actor('username/actor-name');
+// Fetches the john-doe/my-actor object from the API.
+const myActor = await actorClient.get();
+// Starts the run of john-doe/my-actor and returns the Run object.
+const myActorRun = await actorClient.start();
+```
+
+### Nested clients
+
+Sometimes clients return other clients. That's to simplify working with nested collections, such as runs of a given Actor.
+
+```js
+const { ApifyClient } = require('apify-client');
+
+const apifyClient = new ApifyClient({ token: 'MY-APIFY-TOKEN' });
+
+const actorClient = apifyClient.actor('username/actor-name');
+const runsClient = actorClient.runs();
+// Lists the last 10 runs of your Actor.
+const { items } = await runsClient.list({
+    limit: 10,
+    desc: true
+});
+
+// Select the last run of your Actor that finished
+// with a SUCCEEDED status.
+const lastSucceededRunClient = actorClient.lastRun({ status: 'SUCCEEDED' });
+// Fetches items from the run's dataset.
+const { items } = await lastSucceededRunClient.dataset()
+    .listItems();
+```
+
+The quick access to `dataset` and other storage directly from the run client can be used with the [`lastRun()`](https://docs.apify.com/api/client/js/reference/class/ActorClient#lastRun) method.
+
+## Features
+
+Based on the endpoint, the client automatically extracts the relevant data and returns it in the expected format. Date strings are automatically converted to `Date` objects. For exceptions, the client throws an [`ApifyApiError`](https://docs.apify.com/api/client/js/reference/class/ApifyApiError), which wraps the plain JSON errors returned by API and enriches them with other contexts for easier debugging.
+
+```js
+const { ApifyClient } = require('apify-client');
+
+const apifyClient = new ApifyClient({ token: 'MY-APIFY-TOKEN' });
+
+try {
+   const { items } = await client.dataset("non-existing-dataset-id").listItems();
+} catch (error) {
+   // The error is an instance of ApifyApiError
+   const { message, type, statusCode, clientMethod, path } = error;
+   // Log error for easier debugging 
+   console.log({ message, statusCode, clientMethod, type });
+}
+```
+
+### Retries with [exponential backoff](https://docs.apify.com/api/client/js/docs/features#retries-with-exponential-backoff)
```

Comment:
this link actually feels like it points to itself? the features page is now removed by this PR and the content you are linking is exactly here (unless i am missing something)